components/camel-micrometer/src/main/docs/micrometer-component.adoc

```diff
@@ -73,8 +73,12 @@ You should define a dedicated registry by providing
 a `MeterRegistry` bean. Micrometer registries primarily determine the backend monitoring system
 to be used. A `CompositeMeterRegistry` can be used to address more than one monitoring target.
 
-For example using Spring Java Configuration:
+For example using Java Configuration:
```

Comment:
The camel-spring-javaconfig has been removed and this example is not valid anymore

Reply:
I'll remove the whole example

Reply:
I removed example for this configuration. Is it OK now?